src/riak_cs_console.erl

```diff
@@ -62,6 +63,86 @@ cluster_info([OutFile]) ->
             error
     end.
 
+%% @doc Audit bucket ownership by comparing information between
+%%      users bucket and buckets bucket.
+audit_bucket_ownership(_Args) when is_list(_Args) ->
+    audit_bucket_ownership().
+
+audit_bucket_ownership() ->
+    {ok, RcPid} = riak_cs_riak_client:start_link([]),
+    try
+        Inconsistencies = audit_bucket_ownership0(RcPid),
+        io:format("~p", [Inconsistencies])
+    after
+        riak_cs_riak_client:stop(RcPid)
+    end.
+
+%% Construct two sets of {User, Bucket} tuples and
+%% return differences of subtraction in both directions, also output logs.
+audit_bucket_ownership0(RcPid) ->
+    FromUsers = ownership_from_users(RcPid),
+    {FromBuckets, OwnedBy} = ownership_from_buckets(RcPid),
+    lager:debug("FromUsers: ~p~n", [lists:usort(gb_sets:to_list(FromUsers))]),
+    lager:debug("FromBuckets: ~p~n", [lists:usort(gb_sets:to_list(FromBuckets))]),
+    lager:debug("OwnedBy: ~p~n", [lists:usort(gb_trees:to_list(OwnedBy))]),
+    Inconsistencies0 =
+        gb_sets:fold(
+          fun ({U, B}, Acc) ->
+                  lager:info(
```

Comment:
By discussion in chat, output to console may be very noisy when large number of inconsistencies.